examples/EagerMode/QuickStart/README.md

```diff
@@ -0,0 +1,32 @@
+## Dygraph Quick Start
+Train an agent with PARL to solve the CartPole problem, a classical benchmark in RL. Dygraph version of [QuickStart][origin]
+
+## How to use
+### Dependencies:
+
++ [paddlepaddle>=1.5.1](https://github.com/PaddlePaddle/Paddle)
```

Comment:
Does paddle 1.5.1 support eager mode?

Reply:
> Does paddle 1.5.1 support eager mode?

Yes. My paddlepaddle version is just 1.5.1